package.json

```diff
@@ -313,6 +322,11 @@
           "when": "view == projectExplorer && viewItem =~ /^source.*/",
           "group": "inline@0"
         },
+        {
+          "command": "vscode-ibmi-projectexplorer.projectExplorer.iprojShortcut",
+          "when": "view == projectExplorer && viewItem =~ /^source.*/",
```

Comment:
This action should actually be on the project tree item and not the source so this should be `viewItem =~ /^project.*/"`